### ISSUE TITLE
Updating typedoc-plugin-mdn-links from 3.1.0 to 3.1.4 breaks the app 

### ISSUE THREAD

**Issue by ocavue** [Gerrit0/typedoc-plugin-mdn-links#15]
Minimal reproduction ([git repo link](https://github.com/issueset/typedoc-plugin-mdn-links-314)): 

```
$ git clone https://github.com/issueset/typedoc-plugin-mdn-links-314.git 
$ cd typedoc-plugin-mdn-links-314
$ pnpm install
$ pnpm run build 


> pkg-name@0.0.0 build /private/tmp/typedoc-plugin-mdn-links-314
> typedoc

[info] Loaded plugin typedoc-plugin-mdn-links
[info] Converting project at ./packages/pkg1
TypeDoc exiting with unexpected error:
TypeError: app.options.getValue is not a function or its return value is not iterable
    at /private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc-plugin-mdn-links@3.1.4_typedoc@0.25.3/node_modules/typedoc-plugin-mdn-links/dist/index.js:39:28
    at _classThis.resolveExternalLink (/private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc@0.25.3_typescript@5.2.2/node_modules/typedoc/dist/lib/converter/converter.js:238:34)
    at _classThis.resolveLinks (/private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc@0.25.3_typescript@5.2.2/node_modules/typedoc/dist/lib/converter/plugins/LinkResolverPlugin.js:103:54)
    at _classThis.onResolve (/private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc@0.25.3_typescript@5.2.2/node_modules/typedoc/dist/lib/converter/plugins/LinkResolverPlugin.js:85:18)
    at triggerEvents (/private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc@0.25.3_typescript@5.2.2/node_modules/typedoc/dist/lib/utils/events.js:191:43)
    at triggerApi (/private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc@0.25.3_typescript@5.2.2/node_modules/typedoc/dist/lib/utils/events.js:167:13)
    at eventsApi (/private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc@0.25.3_typescript@5.2.2/node_modules/typedoc/dist/lib/utils/events.js:60:18)
    at _classThis.trigger (/private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc@0.25.3_typescript@5.2.2/node_modules/typedoc/dist/lib/utils/events.js:389:13)
    at _classThis.resolve (/private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc@0.25.3_typescript@5.2.2/node_modules/typedoc/dist/lib/converter/converter.js:336:18)
    at _classThis.convert (/private/tmp/typedoc-plugin-mdn-links-314/node_modules/.pnpm/typedoc@0.25.3_typescript@5.2.2/node_modules/typedoc/dist/lib/converter/converter.js:193:18)
 ELIFECYCLE  Command failed with exit code 6.
```

If I downgrade `typedoc-plugin-mdn-links` to v3.1.0 (by editing the `pnpm.overrides.typedoc-plugin-mdn-links` to `"3.1.0"` in the root `package.json` file and run `pnpm install` again), this error won't occur.

**Comment by WikiRik** [comment]
This occurs on the sequelize repo as well; https://github.com/sequelize/sequelize/pull/16779
We are using typedoc 0.25.2 though, but get the same error

**Comment by Gerrit0** [comment]
Shoot, this is https://github.com/TypeStrong/typedoc/commit/c86f4a9a5c326cc5c8b4711e9aaad86fd807d20e, very busy work week for me... I'll try to get a typedoc release out with the fix... Thursday?

**Comment by Gerrit0** [comment]
Should be fixed with typedoc 0.25.4
